Dockerfile

```diff
@@ -166,4 +166,11 @@ RUN tar xf /opt/ort.tar -C /opt/ort --exclude="*.bat" --strip-components 1 && \
 COPY --from=build /usr/local/src/ort/helper-cli/build/scripts/orth /opt/ort/bin/
 COPY --from=build /usr/local/src/ort/helper-cli/build/libs/helper-cli-*.jar /opt/ort/lib/
 
+# Install swift for SPM analysis
+ENV SWIFT_VERSION="swift-5.5.3-RELEASE-ubuntu20.04"
+ENV SWIFT_DOWNLOAD_PATH="swift-5.5.3-release/ubuntu2004/swift-5.5.3-RELEASE/${SWIFT_VERSION}.tar.gz"
+RUN curl "https://download.swift.org/${SWIFT_DOWNLOAD_PATH}" -o swift.tar.gz && \
+    tar -xvzf swift.tar.gz && mv ${SWIFT_VERSION} /usr/local/bin/swift && rm swift.tar.gz
+ENV PATH="/usr/local/bin/swift/usr/bin:${PATH}"
```

Comment:
Please move the version definition to the already existing `Package manager versions` definitions (line 49-63) and only use the actual package manager version without the Ubuntu version: `SWIFT_VERSION=5.5.3`

The definition of the `SWIFT_DOWNLOAD_PATH` is not really necessary, as it is not reused anywhere else.
The actual download of swift should be moved to the `Custom install commands` section of this file (line 122-152) as:
```
curl "https://download.swift.org/swift-${SWIFT_VERSION}-release/ubuntu2004/swift-${SWIFT_VERSION}-RELEASE/swift-${SWIFT_VERSION}-RELEASE-ubuntu20.04.tar.gz
```